src/Bridge/Exporter/AdminExporter.php

```diff
@@ -45,7 +45,7 @@ public function getAvailableFormats(AdminInterface $admin): array
     {
         $adminExportFormats = $admin->getExportFormats();
 
-        if (null !== $adminExportFormats) {
+        if (!empty($adminExportFormats)) {
```

Comment:
```suggestion
        if ([] !== adminExportFormats) {
```

Personally I'm not a fan of `empty`...

Reply:
I used `empty` because it's what I've seen we've been using and because in this particular case `$adminExportFormats` is an `array` so `empty` is clear what is doing, but I don't have a strong opinion on that.

Reply:
Not a fan either, please remove it.

> I used empty because it's what I've seen we've been using 

There is a lot we can improve in the codebase I'm afraid :sweat_smile: